assets/src/edit-story/elements/image/editCropMovable.js

```diff
@@ -151,6 +154,7 @@ EditCropMovable.propTypes = {
   offsetY: PropTypes.number.isRequired,
   imgWidth: PropTypes.number.isRequired,
   imgHeight: PropTypes.number.isRequired,
+  transformFlip: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
```

Comment:
How can it be an object? And how would that work with the `croppedImage.style.transfor` application above

Reply:
There are generally some things broken right now with the Movables + flipping, just looking into this at this moment...